main.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+)
+
+// TODO: add language flag diversion, unit unit test
+
+func main() {
+	// peeper subcommand "get"
+	getCmd := flag.NewFlagSet("get", flag.ExitOnError)
+	// Flags for "get" subcommand
+	// ("flag name", default, "Flag explanation")
+	getAll := getCmd.Bool("all", false, "Get full shortcut list")
+	getByKeyword := getCmd.String("keyword", "", "Find a shortcut with keyword ")
```

Comment:
Presumably these will have to be mutually exclusive, right? In that case, maybe one should be the default, and one should be the flag that trigger special behaviour.

Reply:
Thank you will change so!